src/index.js

```diff
@@ -173,6 +173,21 @@ function EvolvClient(options) {
    */
   this.emit = function(type, metadata, flush) {
     context.pushToArray('events', {type: type,  timestamp: (new Date()).getTime()});
+
+    context.set('fired_events.'+type, true, true);
+    const firedEvents = typeof sessionStorage !== 'undefined' ? sessionStorage.getItem('fe') : '';
+    let eventsList = {};
+    if(firedEvents) {
```

Comment:
@mattstrom is correct, this should be agnostic to node vs browser. Asset Manager isn't -- please move the code responsible for storing the fired_events in session storage and re-hydrating the local context to the ass3et manager.

We have a listener for context updates and fired events that can be used